app/src/main/java/ro/code4/monitorizarevot/ui/login/LoginViewModel.kt

```diff
@@ -34,10 +32,11 @@ class LoginViewModel : BaseViewModel() {
         getFirebaseToken(phone, password)
     }
 
-    private fun onSuccessfulLogin(loginResponse: LoginResponse, firebaseToken: String) {
+    private fun onSuccessfulLogin(loginResponse: LoginResponse) {
         logD("onSuccessfulLogin")
+
         sharedPreferences.saveToken(loginResponse.accessToken)
-        registerForNotification(firebaseToken)
+        onSuccessfulRegisteredForNotification()
     }
 
     private fun onSuccessfulRegisteredForNotification() {
```

Comment:
the `onSuccessfulRegisteredForNotification` method can now be merged with `onSuccessfulLogin` as there's no need for separate methods :)

Reply:
Done